opty/direct_collocation.py

```diff
@@ -133,10 +133,16 @@ class ConstraintCollocator(object):
     """
 
     time_interval_symbol = sym.Symbol('h', real=True)
+    expeirment_data_symbol = sym.Symbol('e', real=True)
```

Comment:
Thanks for the check, I will adjust them.

On Thu, May 19, 2016 at 2:20 PM, Jason K. Moore <notifications@github.com>
wrote:

> In opty/direct_collocation.py
> <https://github.com/csu-hmc/opty/pull/40#discussion_r63928902>:
>
> > @@ -133,10 +133,16 @@ class ConstraintCollocator(object):
> >      """
> >
> >      time_interval_symbol = sym.Symbol('h', real=True)
> > +    expeirment_data_symbol = sym.Symbol('e', real=True)
>
> spelling
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/csu-hmc/opty/pull/40/files/2a129a976a12bdb95e64d7afaecd93fcc78afcaf#r63928902>
>